src/init/consented/comscore.ts

```diff
@@ -16,6 +16,9 @@ const getGlobals = (keywords: string): ComscoreGlobals => {
 		c1: comscoreC1,
 		c2: comscoreC2,
 		cs_ucfr: '1',
+		options: {
```

Comment:
I think we also need to be passing a 1st party ID up to comscore as `cs_fpid`. Talking to @Amouzle we could use the [PPID](https://github.com/guardian/commercial/blob/33985dce5c80dc2bd925f5c61298c242ded2baff/src/init/consented/prepare-googletag.ts#L23)

Reply:
The documentation mentions that it's automatically collected?

Reply:
@Jakeii Do you know how how they would automatically collect the PPID?

Reply:
Ah I think there should be a distinction here:
`cs_fpid`: The first party identifier added by Comscore, this is the mandatory change requested by ComScore by end of March. 
`cs_fpit` (optional): This would be our own identity ID (like we use for PPID) which we would need to populate, allowing ComScore to deduplicate unique users when that data is available. 

There was an ask from the US for us to include `cs_fpit` in this work. However, for expediency, I think we should roll out without this for now and we can add this as a follow up (as I'm not sure all stakeholders are on the same page about the necessity of this anyhow).

Reply:
(deleted my last comment, I confused my IDs, apologies). Will chat with you @Jakeii today

Reply:
@mchv Upon further inspection `cs_fpid` is just the value of the cookie `_scor_uid`
<img width="1388" alt="Screenshot 2024-03-22 at 10 39 35" src="https://github.com/guardian/commercial/assets/1731150/7cec5a61-367c-47a4-8e23-6e7d60c721ef">

Will chat with @SiAdcock and @Amouzle to confirm what we need to pass in

Reply:
@mchv, and @SiAdcock so it doesn't collect the PPID automatically.